docs/operator-guide/cd/manual-approval.md

```diff
@@ -0,0 +1,123 @@
+# Manual Approval in Pipelines
+
+The manual approval feature gives users a smooth and controlled process of promoting applications from lower, non-critical environments, such as development or QA, to mission-critical environments like production. Additionally, it ensures that only thoroughly tested and verified changes are deployed to production, minimizing the risk of causing errors or instability.
```

Comment:
we also need to add as pre-requisites that https://github.com/kuberocketci/tekton-custom-task is installed